### PR TITLE
77749 List comments

### PR DESCRIPTION
**Pull request by bratli**
DevOps#77749

### CODE REVIEW

**Comment by bratli** [comment]
@hbjorgo is looking at the failing tests. Will wait to merge until these are fixed.

File: src/Equinor.ProCoSys.IPO.Query/GetComments/GetCommentsQuery.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using MediatR;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.GetComments
+{
+    public class GetCommentsQuery : IRequest<Result<List<CommentDto>>>, IInvitationQueryRequest
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.Query/GetComments/GetCommentsQuery.cs:7]: Missing tests for AccessValidator

File: src/Equinor.ProCoSys.IPO.Query/GetComments/GetCommentsQueryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            var invitation = await
+                (from inv in _context.QuerySet<Invitation>()
+                        .Include(i => i.Comments)
+                        .Where(i => i.Id == request.InvitationId)
+                 select inv).SingleOrDefaultAsync(token);
+
+            if (invitation == null)
+            {
+                return new NotFoundResult<List<CommentDto>>($"Entity with ID {request.InvitationId} not found");
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.Query/GetComments/GetCommentsQueryHandler.cs:33]: return new NotFoundResult<InvitationDto>(Strings.EntityNotFound(nameof(Invitation), request.InvitationId));
Search for NotFoundResult in this project and you'll see other statements as this which should use Strings.EntityNotFound

**Review by janid1967 — APPROVED** [review]
Approved w suggestions

**Pull request merged**